server/methods.ts

```diff
@@ -191,51 +191,17 @@ Meteor.methods({
     }
   },
   /**
-   * Increment the number of likes a review has gotten by 1.
-   *
-   * Returns 1 on success
-   * Returns 0 on error
+   * Deprecated for security reasons
    */
   async incrementLike(id) {
```

Comment:
I don't think this is used anywhere (since you updated the front end with axios calls in this PR too) so we can remove this function (unless you want to keep it for book keeping or something).